typing/typecore.ml

```diff
@@ -7212,27 +7215,45 @@ let report_error ~loc env = function
       Location.errorf ~loc
         "The instance variable %a is overridden several times"
         Style.inline_code v
-  | Coercion_failure (ty_exp, err, b) ->
+  | Coercion_failure(ty_exp, err, false) ->
```

Comment:
I am allergic to raw booleans. Have you considered moving `Coercion_failure` to an inline record, or using a dedicated variant type?